tasks/section2.yml

```diff
@@ -359,7 +573,14 @@
       - rule_2.2.7
 
 - name: "SCORED | 2.2.7 | PATCH | Ensure NFS and RPC are not enabled"
-  command: /bin/true
+  service:
+      name: "{{ item }}"
+      state: stopped
+      enabled: no
+  with_items:
+  - nfs
+  - rpcbind
```

Comment:
Sorry I was speaking directly about the with_items section of this task
```
  with_items:
  - nfs
  - rpcbind
```
should be:

```
  with_items:
      - nfs
      - rpcbind
````